### PR TITLE
ENG-2185: home page redesign and quickstarts landing page

### PR DESCRIPTION
**Pull request by brigonzalez**
## Description

- home page redesign and quickstarts landing page

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):
Screen Shot 2022-02-11 at 4 26 01 PM

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

**Comment by brigonzalez** [comment]
getting a weird issue where the icons aren't appearing on the quickstart home page. wondering if this is just a me problem or if others can reproduce this issue

**Review by armsteadj1 — APPROVED** [review]

File: index.markdown [hunk 1]
```diff
@@ -18,90 +18,118 @@ image:
...
-                        </div>
-                    </div>
-                </a>
+    <body class="home-page">
+        <h1>Documentation</h1>
+        <h5>Explore our APIs, SDKs, and all of the tools you need to make your sensitive data more secure and usable.</h5>
+        <div class="quickstarts">
+            <div class="quickstart-info">
+                <div class="quickstart-header">Quickstarts</div>
+                <div class="quickstart-subtitle">Learn how to use Basis Theory use tokenization to encrypt personal information and safely share it to Auth0.</div>
```
Comment (armsteadj1) [inline index.markdown:27]:
```suggestion
                <div class="quickstart-subtitle">Get a foundational understanding of how to secure your data with Basis Theory allowing you to no longer worry about the underlying storage or encryption of the data.</div>
```

File: index.markdown [hunk 2]
```diff
@@ -18,90 +18,118 @@ image:
...
-                <a href="/guides/collect-pii-js/">
-                    <div class="card">
-                        <img src="./assets/images/card4.svg">
-                        <div class="container">
-                            <h4>Collect Customer (PII) Data</h4>
-                            <p>Basis Theory is intended to handle any payload of data you can serialize. To show the flexibility of our tokenization platform, the following guide will show you how you can easily tokenize your customer's PII (Personally Identifiable Information) data to keep it safe and secure.</p>
-                        </div>
+                </div>
+                <div class="card-title">What are Tenants?</div>
+                <div>Where your Tokens and Applications are defined, managed, and stored.</div>
```
Comment (armsteadj1) [inline index.markdown:59]:
```suggestion
                <div>Tenats are where Tokens and Applications are defined, managed, and stored.</div>
```

File: index.markdown [hunk 3]
```diff
@@ -18,90 +18,118 @@ image:
...
-                </a>
-                <a href="/guides/setup-your-first-reactor/">
-                    <div class="card">
-                        <img src="./assets/images/card4.svg">
-                        <div class="container">
-                            <h4>Setup your first Reactor</h4>
-                            <p>Token Reactors are foundational to your ability to make your data more usable within your application.</p>
+                </div>
+                <div class="card-title">What is the Proxy?</div>
+                <div>Send tokenized data without touching it.</div>
```
Comment (armsteadj1) [inline index.markdown:70]:
```suggestion
                <div>Proxy enables making HTTP requests containing sensitive tokenized data to your 3rd party partners, without your systems needing access to the raw data.</div>
```

**Review by armsteadj1 — CHANGES_REQUESTED** [review]

**Review by armsteadj1 — APPROVED** [review]

**Pull request merged**